File: 1-Review/C/review-1.c
#pragma warning(disable:4996)
#include <stdio.h>

int sum(int i);

int main() {

	int X;
	scanf("%d", &X);

	int total = 0;
	for (int i = 1; i <= X; i++)
		total += sum(i);

	printf("%d", total);

	return 0;
}

int sum(int i) {
	int sum = 0;
	for (int j = 1; j <= i; j++)
		sum += j;

	return sum;
}

File: 1-Review/Python/review-1.py
def sum(i):
    sum = 0
    for j in range (1, i + 1):
        sum += j
    
    return sum

def main():
    X = int(input())
    
    total = 0
    for i in range(1, X + 1):
        total += sum(i)

    print(total)

if __name__ == '__main__':
    main()

    